source/query-with-natural-language/enable-natural-language-querying.txt

```diff
@@ -36,8 +36,6 @@ you must:
 - Sign into Atlas. For details on how to get a free Atlas 
   account and deploy a free M0 cluster, see :ref:`atlas-getting-started`.
 
-.. BEGIN-COMPASS-ONLY
```

Comment:
Removed these tags from this page since Atlas docs will have it's own version of this page that isn't migrated from Compass.